node/src/GlideClient.ts

```diff
@@ -609,4 +613,95 @@ export class GlideClient extends BaseClient {
     public publish(message: string, channel: string): Promise<number> {
         return this.createWritePromise(createPublish(message, channel));
     }
+
+    /**
+     * Sorts the elements in the list, set, or sorted set at `key` and returns the result.
+     *
+     * The `sort` command can be used to sort elements based on different criteria and
+     * apply transformations on sorted elements.
+     *
+     * To store the result into a new key, see {@link sortStore}.
+     *
```

Comment:
I am just wondering whether you should add this:

```suggestion
     * See https://valkey.io/commands/sort/ for more details.
```